### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by turiot** [Wisser/Jailer#84] [General]
Many thanks for your tool, so useful to explore unknown schema.
It's also a delight for swing fans.
And a good idea to ignore reddit, full of unproductive critics.

**Comment by armandino** [comment]
Couldn't agree more. Came here to say that it's a great application. Really good for exploring a schema and as an SQL client. Many thanks for such an awesome tool!

**Comment by canine-afficionado** [comment]
+1 from me also - just found this tool and it's amazing for an open source tool. No open issues either! Thanks for all your hard work.

**Comment by schuerg** [comment]
@Wisser 

Thank you for this excellent tool - it significantly reduced the manual steps in my workflow. Your work is greatly appreciated!
